ios-runtime/SdkgenRuntime.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |spec|  
+  spec.name                    = 'SdkgenRuntime'
+  spec.version                 = '0.1.0'
+  spec.summary                 = 'Cubos sdkgen.'
```

Comment:
"sdkgen is a tool to help design, implement and maintain good APIs with minimal effort"